conans/test/integration/environment/test_env.py

```diff
@@ -491,3 +491,39 @@ def generate(self):
     assert 2 == str(out).count("Restoring environment")
     assert "VAR1=!!" in out
     assert "VAR2=!!" in out
+
+
+@pytest.mark.skipif(platform.system() != "Windows", reason="Path problem in Windows only")
+@pytest.mark.parametrize("num_deps", [3, 60])
```

Comment:
Leave this test out of the new PS functionality, this is still not working, and not addressed yet.

Reply:
Done